etc/config/slang.amazon.properties

```diff
@@ -1,16 +1,18 @@
 compilers=&defaultslangc
 
-defaultCompiler=slangc_2025_2_2
+defaultCompiler=slangc_2025_6_1
 supportsBinary=false
 compilerType=slang
 instructionSet=spirv
 
 disassemblerPath=/opt/compiler-explorer/SPIRV-Tools-master/build/tools/spirv-dis
 
-group.defaultslangc.compilers=slangc_2024_14_6:slangc_2025_2_2
+group.defaultslangc.compilers=slangc_2024_14_6:slangc_2025_2_2:slangc_2025_6_1
 group.defaultslangc.versionFlag=-version
 
 compiler.slangc_2024_14_6.exe=/opt/compiler-explorer/slang-2024.14.6/bin/slangc
 compiler.slangc_2024_14_6.name=slangc 2024.14.6
 compiler.slangc_2025_2_2.exe=/opt/compiler-explorer/slang-2025.2.2/bin/slangc
 compiler.slangc_2025_2_2.name=slangc 2025.2.2
+compiler.slangc_2025_6_1.exe=/opt/compiler-explorer/slang-2025.6.1/bin/slangc
```

Comment:
This seems to be the wrong path. After installing your infra PR:
```
$ ls -ld /opt/compiler-explorer/slang-202*
drwxr-xr-x 6 root root 6144 Dec  2 10:36 /opt/compiler-explorer/slang-2024.14.6
drwxr-xr-x 7 root root 6144 Jan 24 17:07 /opt/compiler-explorer/slang-2025.2.2
drwxr-xr-x 7 root root 6144 Mar 27 16:20 /opt/compiler-explorer/slang-2025.6.2
```

Maybe this is a `.2` release?

Reply:
ugh, yep, let me update this to `.2`

Reply:
funny thing had it, just forgot to push up here when I updated the infra to `.2`, should be good now (thanks as always!)